examples/hello-world/go.mod

```
@@ -0,0 +1,201 @@
+module hello-world
+
+go 1.21.1
+
+toolchain go1.21.3
+
+require (
+	github.com/hashicorp/go-plugin v1.5.2
+	github.com/ignite/cli/v28 v28.0.0
```

Ignite dependency could be updated to the latest so it includes support for the new `.ign` app extension:

```suggestion
	github.com/ignite/cli/v28 v28.1.1
```

---

Fixed